pkgs/tools/inputmethods/hime/default.nix

```
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, pkgconfig, which, gtk2, gtk3, qt4, qt5, libXtst,
+libchewing }:
+
+stdenv.mkDerivation {
+  name = "hime";
+
+  src = fetchFromGitHub {
+    owner = "hime-ime";
+    repo = "hime";
+    rev = "c89751a58836906e6916355fd037fc74fd7a7a15";
+    sha256 = "024w67q0clzxigsrvqbxpiy8firjvrqi7wbkkcapzzhzapv3nm8x";
+  };
+
+  nativeBuildInputs = [ which pkgconfig ];
+  buildInputs = [ libXtst libchewing gtk2 gtk3 qt4 qt5.qtbase ];
+
+  preConfigure = ''
+    patchShebangs configure
+  '';
+  configureFlags = [ "--disable-lib64" "--disable-qt5-immodule" ];
+
+
+  meta = with stdenv.lib; {
+    homepage      = "http://hime-ime.github.io/";
+    downloadPage  = "https://github.com/hime-ime/hime/downloads";
+    description   = "A useful input method engine for Asia region";
+    license       = licenses.gpl2Plus;
```

the readme says: "LGPLv2.1 (Qt immodules are GPLv2)"
```suggestion
    license       = with licenses; [ lgpl21Plus gpl2Plus ];
```

---

Thanks for review